sdf_timing/utils.py

```diff
@@ -1,3 +1,42 @@
+import re
+
+
+def get_scale(timescale):
+    """Convert sdf timescale to scale factor
+
+    >>> get_scale('1.0 fs')
+    1e-15
+
+    >>> get_scale('1ps')
+    1e-12
+
+    >>> get_scale('10 ns')
+    1e-08
+
+    >>> get_scale('10.0 us')
+    9.999999999999999e-06
+
+    >>> get_scale('100.0ms')
+    0.1
+
+    >>> get_scale('100 s')
```

Comment:
Can the scale string be in other floating point forms? (IE scientific?)

Reply:
No the format is pretty strict in the SDF spec